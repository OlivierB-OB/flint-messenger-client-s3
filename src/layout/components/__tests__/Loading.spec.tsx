import React from 'react';
import { shallow } from 'enzyme';
import { Loading } from '../Loading';

describe('Loading', () => {
  it('should display a loader', async () => {
    const component = shallow(<Loading />);
    expect(component).toMatchSnapshot();
  });
});
