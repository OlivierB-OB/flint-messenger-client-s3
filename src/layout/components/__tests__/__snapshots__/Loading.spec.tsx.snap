// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Loading should display a loader 1`] = `
<WithStyles(ForwardRef(Grid))
  alignItems="center"
  container={true}
  direction="column"
  justify="center"
  style={
    Object {
      "height": "50vh",
    }
  }
>
  <WithStyles(ForwardRef(Grid))
    item={true}
  >
    <WithStyles(ForwardRef(CircularProgress)) />
  </WithStyles(ForwardRef(Grid))>
  <WithStyles(ForwardRef(Grid))
    item={true}
  >
    Loading...
  </WithStyles(ForwardRef(Grid))>
</WithStyles(ForwardRef(Grid))>
`;
